ffmpeg_progress_yield/__main__.py

```diff
@@ -40,7 +46,10 @@ def main() -> None:
         for progress in ff.run_command_with_progress():
             print(f"{progress}/100")
 
-    print(ff.stderr)
+    if args.progress_only:
+        print("\x1B[K")
```

Comment:
What is the purpose of this?

With it enabled, and without it, these are the outputs I get (macOS, Bash):

<img width="805" alt="image" src="https://github.com/user-attachments/assets/bd0b222d-e896-40d4-bdc8-ceb61cf42f19">

To me it seems that not printing this escape sequence is the better option?